src/fsharp/IlxGen.fs

```diff
@@ -6393,16 +6390,15 @@ and GenTypeDef cenv mgbuf lazyInitInfo eenv m (tycon:Tycon) =
                   yield
                       { Name          = ilFieldName
                         Type          = ilPropType
-                        IsStatic      = isStatic
+                        Attributes    = (if isStatic then FieldAttributes.Static else FieldAttributes.Private) ||| 
+                                        (if ilFieldName="value__" && tycon.IsEnumTycon then FieldAttributes.SpecialName else FieldAttributes.Private) |||
+                                        (if ilNotSerialized then FieldAttributes.NotSerialized else FieldAttributes.Private) |||
```

Comment:
likewise here.  Would prefer ``enum<_>(0)``

Reply:
Dotnet framework enum guidance suggests a Zero value.

https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/enum

Which might be a preferable approach.
<snip>
✓ DO provide a value of zero on simple enums. 
</snip>

Reply:
I thought about it but after I worked on ILMethodDef I also merge the Access to the the Attributes and it solve me that there is no None on this enum I'll push that when I'll finish